src/components/project/home-block.vue

```diff
@@ -0,0 +1,140 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div class="project-home-block">
+    <div class="title">
+      <router-link :to="projectPath(project.id)">{{ project.name }}</router-link>
+      <span v-if="project.keyId" class="encrypted badge"
+        :title="$t('encryptionTip')">
+        <span class="icon-lock project-icon"></span>
+        {{ $t('encrypted') }}
+        </span>
+    </div>
+    <table v-if="visibleForms != null" class="project-form-table table">
+      <project-form-row v-for="form of visibleForms" :key="form.xmlFormId" :form="form"/>
+    </table>
+    <div v-if="showExpander">
+      <span class="expand-button" @click.prevent="toggleExpanded">
+        <template v-if="!expanded">
+          {{ $tcn('showMore', project.formList.length) }}<span class="icon-angle-down"></span>
+        </template>
+        <template v-else>
+          {{ $tcn('showFewer', project.formList.length) }}
+        </template>
+      </span>
+    </div>
+  </div>
+</template>
+
+<script>
+import routes from '../../mixins/routes';
+
+import ProjectFormRow from './form-row.vue';
+
+export default {
+  name: 'ProjectHomeBlock',
+  components: { ProjectFormRow },
+  mixins: [routes()],
+  props: {
+    project: {
+      type: Object,
```

Comment:
I know I just said to specify `Object` as the prop type instead of `Project`, but I actually figured out a way for us to continue importing presenter classes (in 359a8c650292b437c3095c2825a672a0ecfd6463). That means that it actually does work to specify `Project` as the prop type here.